### PR TITLE
MAP-1729 - Update the service from the template project

### PR DESCRIPTION
**Pull request by Thource**
This PR brings the project up to date with hmpps-template-typescript, see https://dsdmoj.atlassian.net/browse/MAP-1729

### CODE REVIEW

**Review by Mjwillis — APPROVED** [review]
Lgtm

File: assets/js/multiSelect.js [hunk 1]
```diff
@@ -13,17 +13,32 @@ function updateCheckedCount() {
...
+    return (
+      '' +
+      '<div class="govuk-checkboxes__item govuk-checkboxes--small moj-multi-select__checkbox">' +
+      '  <input type="checkbox" class="govuk-checkboxes__input" id="checkboxes-all" autocomplete="off">' +
+      '  <label class="govuk-label govuk-checkboxes__label moj-multi-select__toggle-label" for="checkboxes-all">' +
+      '    <span class="govuk-visually-hidden">Select all</span>' +
+      '  </label>' +
+      '</div>'
+    )
+  }
```
Comment (jimbali) [inline assets/js/multiSelect.js:28]: I had to add this monkey patch to get it to clear the select all checkbox, but I wasn't able to write an integration test that properly tests it. Let me check if it still works...

Comment (jimbali) [reply assets/js/multiSelect.js:28]: Yep still seems to work 👍

**Review by jimbali — APPROVED** [review]

File: Dockerfile [hunk 2]
```diff
@@ -53,16 +54,13 @@ COPY --from=build --chown=appuser:appgroup \
...
-
 COPY --from=build --chown=appuser:appgroup \
         /app/dist ./dist
 
 COPY --from=build --chown=appuser:appgroup \
         /app/node_modules ./node_modules
 
-EXPOSE 3000 3001
+EXPOSE 3000
 ENV NODE_ENV='production'
```
Comment (marcusaleman) [inline Dockerfile:64]: @Thource this var is duplicated

**Pull request merged**